bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+exec foreman start -f Procfile.dev "$@"
```

Comment:
Do you need this file?

Reply:
No, I needed, maybe this file generate, when i try install bootstrap